site/src/i18n/index.ts

```diff
@@ -0,0 +1 @@
+export * from "./i18n"
```

Comment:
[barrel file](https://adrianfaciu.dev/posts/barrel-files/) FTW 🚀

Reply:
Yeah, I figure this is a good place for one.